server/services/player.ts

```diff
@@ -7,17 +7,19 @@ import {
 import crypto from "crypto"
 
 export const findOrCreatePlayer = (publicKey: string) => {
-  const existingPlayer = store.players.find(player => player.publicKey === publicKey)
+  const existingPlayer = store.players.find(
+    (player) => player.publicKey === publicKey,
+  )
 
   if (existingPlayer) return existingPlayer
 
-  const hash = crypto.createHash('md5')
+  const hash = crypto.createHash("md5")
   hash.update(publicKey)
-  
+
   const player: Player = {
     id: store.players.length,
     publicKey,
-    username: `1_${hash.digest('hex').slice(0, 6)}`,
+    username: `1_${hash.digest("hex").slice(0, 6)}`,
```

Comment:
Feel free to add an eslint rule for quotes. Double or single are okay.

Reply:
I'd say double because they're used almost everywhere else in this project.